sqlglot/tokens.py

```diff
@@ -986,13 +986,17 @@ class Tokenizer(metaclass=_Tokenizer):
         "_peek",
         "_prev_token_line",
         "_rs_dialect_settings",
+        "use_rs_tokenizer",
     )
 
     def __init__(self, dialect: DialectType = None) -> None:
         from sqlglot.dialects import Dialect
 
         self.dialect = Dialect.get_or_raise(dialect)
 
+        # initialize `use_rs_tokenizer`, and allow it to be overwritten per Tokenizer instance
+        self.use_rs_tokenizer = USE_RS_TOKENIZER
```

Comment:
can you pass it into the iniatializer?

use_rs_tokenizer: bool = USE_RS_TOKENIZER

then set it here on 998

Reply:
Done @tobymao - updated